Documentation/Developer/Index.rst

```diff
@@ -59,44 +59,44 @@ Enter the basic meta data for your extension in the left panel of the modeler:
 |**Key**        |The extension key is a unique identifier for your extension                                                                |
 +---------------+---------------------------------------------------------------------------------------------------------------------------+
-|**Description**|Here you can describe in more detail what your extension does                                                              |
+|**Description**|More details what your extension does                                                                                      |
 +---------------+---------------------------------------------------------------------------------------------------------------------------+
-|**Version**    |A good versioning schema helps you to track changes and versions. We recommend `Semantic Versioning <http://semver.org/>`_ |
+|**Version**    |A good versioning schema helps to track changes and versions. We recommend `Semantic Versioning <http://semver.org/>`_     |
 +---------------+---------------------------------------------------------------------------------------------------------------------------+
 |**State**      |The state shows other users if your extension has already reached a stable state. Otherwise it can be alpha or beta.       |
 +---------------+---------------------------------------------------------------------------------------------------------------------------+
-|**Persons**    |Here you can add developers or project managers                                                                            |
+|**Persons**    |Possibility to add developers or project managers                                                                          |
 +---------------+---------------------------------------------------------------------------------------------------------------------------+
 
 
 Adding plugins
 ``````````````
 
 Every extension that should generate output in the frontend needs a plugin. It will be shown in the "plugin type" field
-of the content element "Plugin". You can add it to your extension in the left panel:
+of the content element "Plugin". It can be added to your extension in the left panel:
 
-+--------+--------------------------------------------------------+
-|**Name**|A short, speaking name that is displayed in the backend |
-+--------+--------------------------------------------------------+
-|**Key** |A unique key to identify the plugin in your extension   |
-+--------+--------------------------------------------------------+
++--------+---------------------------------------------------------+
+|**Name**| A short, speaking name that is displayed in the backend |
++--------+---------------------------------------------------------+
+|**Key** | A unique key to identify the plugin in your extension   |
++--------+---------------------------------------------------------+
```

Comment:
Did you render the page? Additional signs shouldn't be required I think, as the lines are markup for rendering only.

Reply:
No, at the moment I have no working solution to check the rendering.
I prefer to have a leading blank. 
I do not want to change all tables.